packages/database/src/migrations/20210214234630-AddDataDownloadNationalFETP-modifies-data.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+import { insertObject, arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const DATA_ELEMENT_CODE = 'FGDS';
```

Comment:
Bit pedantic but I'd rename this variable `SURVEY_CODE`

Reply:
done, thanks!